src/main/java/com/julienviet/pgclient/impl/ArrayTuple.java

```diff
@@ -85,8 +89,34 @@ public Temporal getTemporal(int pos) {
     return (Temporal) get(pos);
   }
 
+  @Override
+  public LocalDate getLocalDate(int pos) {
+    return (LocalDate) get(pos);
```

Comment:
what should we do if the type is not correct ? can we try to convert it ?

Reply:
I think we can check the object type and do the proper conversion if needed